server/routes/classes.ts

```diff
@@ -27,28 +52,72 @@ this is responsible of another microservice
 
 //TODO: Connect to course microservice in frontend
 
-router.post("/", async (req: Request, res: Response) => {
-    const { title, description, order, file }: ClassInputs = req.body
-
-    if (!title || !description || !order || !file) {
-        return res.status(400).json({
-            error: "Missing required fields (title, description, order, file)",
-        })
+router.post("/", upload.single('file'), async (req: Request, res: Response) => {
+    
+    try{
+
+
+        const { title, description, order }: ClassInputs = req.body
+
+        if (!title || !description || !order || !req.file) {
+            return res.status(400).json({
+                error: "Missing required fields (title, description, order, file)",
+            })
+        }
+
+        // Verify file type
+        const contentType = req.file.mimetype;
+
+        if (!allowedMimeTypes.includes(contentType)) {
+            return res.status(400).json({
+                error: "Invalid file type. Only video files are allowed.",
```

Comment:
Maybe it would be correct to put in the message what type of files are accepted (mp4, quicktime and mpeg)